pkg/controller/pod_driver.go

```diff
@@ -780,13 +780,14 @@ func (p *PodDriver) doAbortFuse(mountpod *corev1.Pod, devMinor uint32) error {
 	waitCtx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
 	defer cancel()
 	for {
-		if waitCtx.Err() == context.Canceled {
+		if waitCtx.Err() == context.Canceled || waitCtx.Err() == context.DeadlineExceeded {
 			klog.Errorf("fuse abort job timeout")
```

Comment:
add more info about the job? for example job name?

Reply:
and maybe change log level to "error" in L753, just to put all abnormalities in error level